sdk/openai/inference/src/lib.rs

```diff
@@ -0,0 +1,8 @@
+mod auth;
+pub mod clients;
+mod helpers;
+mod models;
+mod options;
+
+pub use models::*;
```

Comment:
Request models should be flattened into the root, but response models should not. The idea is that we want to expose anything you're probably going to have to type in the root, unless there are a lot of request models. We haven't quantified yet.

So perhaps this would be better:
```suggestion
pub use models::{request::*, response};
```